src/library/Cardano/CoinSelection/Migration.hs

```diff
@@ -184,12 +182,10 @@ selectCoins feeOpts batchSize utxo =
         put rest
         pure batch
 
---------------------------------------------------------------------------------
--- Internal Functions
---------------------------------------------------------------------------------
-
--- Try to find a fixed "ideal" number of input transactions that would generate
+-- | Try to find a fixed "ideal" number of input transactions that would generate
 -- relatively balanced transactions.
+--
+-- @since 1.0.0
 idealBatchSize :: CoinSelectionLimit -> Word8
```

Comment:
Should this `Word8` type be widened?

Related comment: https://github.com/input-output-hk/cardano-coin-selection/issues/17#issuecomment-618161639

Reply:
So long as the `CoinSelectionLimit` remains unchanged, no, since the domain here is a direct consequence of the co-domain. Word8 is arguable small, and I do not imagine a transaction with more than 65535 inputs/ouputs! That would be insane. So if Word16 is an easy win, it could be an option.

Reply:
`Word16` is indeed an easy win. I've created a PR from the above branch here: https://github.com/input-output-hk/cardano-coin-selection/pull/58